### PR TITLE
refactor(meditation): refactor bell + add support for new sound files in AAC

### LINKED ISSUE #499: Stopping a meditation, then starting a new one, breaks the proper timing of the sound alerts.
I started a 19 minute walking + 15 minute sitting meditation.
Then I immediately stopped it.
And afterwards I started a 15 minute walking + 15 minute sitting meditation.

**Expected behavior:**

1. Start of walking meditation.
2. After 15 minutes, the alarm goes off.
3. Start sitting meditation.
4. After 15 minutes, the alarm goes off.

**Actual behavior:**

1. Start of walking meditation.
2. After 15 minutes, the alarm went off.
3. Start sitting meditation.
4. After 4 minutes, the alarm went off.
5. After 11 more minutes, (when the alarm should have sounded), no sound was played. Navigation went back to the list of meditators.
6. A few minutes after, another alarm went off.

Tomorrow I'll try to reproduce this on my local environment, and I'll probably develop a fix myself. Just publishing this note to avoid forgetting.

### PR DESCRIPTION
**Pull request by olastor**
fixes #499 

not tested yet.

### CODE REVIEW

**Comment by binarious** [comment]
Do we still need these hundreds of mp3s?

**Comment by olastor** [comment]
@binarious If you mean the `.ogg` files https://share.sirimangalo.org/static/sounds/ , then yes 😁 I've even added some more  `.m4a` files... It seems like there is no good solution for the bell right now.

**Comment by binarious** [comment]
Maybe we should ask how many people are really using this and how it works for them. It's quite an overhead in my opinion. What do you think?

**Comment by olastor** [comment]
@binarious About 82 people have it activated right now. For longer periods like half an hour the only alternative on mobile phones is to advise the users to take a seperate timer app ... Should we go this way?

**Comment by olastor** [comment]
Or stick to push notifications.

**Comment by binarious** [comment]
I just think we shouldn't add more features to the bell/timer and focus on other things. But this PR looks good nevertheless.

File: src/app/meditation/meditation.component.ts [hunk 1]
```diff
@@ -256,132 +273,83 @@ export class MeditationComponent implements OnInit, OnDestroy {
...
+    // setup timer & bell,
     // prevent 'gesture-requirement-for-media-playback' on mobile browsers
     this.bell = new Audio();
-    this.bell.src = 'assets/audio/halfsec.mp3';
+    this.bell.src = '/assets/audio/halfsec.mp3';
     this.bell.play();
 
-    this.startTimer(walking, sitting);
+    // check for Network API support
+    const connection = window.navigator['connection']    ||
```
Comment (binarious) [inline src/app/meditation/meditation.component.ts]:
```
const connection = window.navigator['connection']    
  || window.navigator['mozConnection'] 
  || null;
```

File: src/app/meditation/meditation.component.ts [hunk 2]
```diff
@@ -23,13 +23,13 @@ import * as StableInterval from 'stable-interval';
 })
 export class MeditationComponent implements OnInit, OnDestroy {
 
+
```
Comment (binarious) [inline src/app/meditation/meditation.component.ts]: Please remove this line.

**Comment by olastor** [comment]
Conflicts should be resolved

**Pull request merged**